### PR TITLE
Replaces Marketplace with External Services section

### PR DESCRIPTION
**Pull request by luciddreamz**
@corey112358 Please merge shortly

### CODE REVIEW

File: lib/en/external-services-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+nav_priority: 1
+meta_desc: Using external services with your OpenShift applications.
+---
+= External Services
+
+[float]
+= External Services
+
+[.lead]
+OpenShift makes it easy to connect external services to your applications. External services can provide a wide range of extra functionality and capabilities for your applications, but are generally categorized into two areas: application services and devops services. For application services, common uses include data persistence, search, caching, messaging, and more. For devops, services provide software lifecycle capabilities such as continuous integration and load testing.
```
Comment (corey112358) [inline lib/en/external-services-overview.adoc]: Change "applications, but are" to "applications, and are"

File: lib/en/external-services-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+[float]
+= External Services
+
+[.lead]
+OpenShift makes it easy to connect external services to your applications. External services can provide a wide range of extra functionality and capabilities for your applications, but are generally categorized into two areas: application services and devops services. For application services, common uses include data persistence, search, caching, messaging, and more. For devops, services provide software lifecycle capabilities such as continuous integration and load testing. 
+
+== Using External Services
+When working with an external resource, we recommend that you use environment variables to model the external service as a native OpenShift link:https://access.redhat.com/documentation/en-US/OpenShift_Online/2.0/html/User_Guide/chap-Cartridges.html#Add-on_Cartridges3[add-on cartridge]. That way your application can connect with the external service the same way as any other internal service.
+
+Below, we outline a framework for connecting to external services to your application along with examples for our trusted partner providers.
```
Comment (corey112358) [inline lib/en/external-services-overview.adoc]: change "connecting to external services" to "connecting external services"

Comment (luciddreamz) [inline lib/en/external-services-overview.adoc]: Resolved.

File: lib/en/external-services-overview.adoc (hunk 1 above)
Comment (luciddreamz) [inline lib/en/external-services-overview.adoc]: Resolved.

**Comment by jsvgoncalves** [comment]
@luciddreamz please fix the merge conflicts

**Comment by luciddreamz** [comment]
@jsvgoncalves Resolved

**Pull request merged**